osidb/models.py

```diff
@@ -1576,7 +1555,9 @@ def _create_flaw(self, *args, **kwargs) -> Flaw:
 
         shared_acl = {"acl_read": self.acl_read, "acl_write": self.acl_write}
         # ensure that a flaw always contains external id (even if BZ sync is disabled)
-        shared_flaw = shared_acl | {"meta_attr": {"external_ids": self.external_id}}
+        shared_flaw = shared_acl | {
+            "meta_attr": {"external_ids": json.dumps([self.external_id])}
+        }
```

Comment:
We should create a migration for this change in the near future to keep the data consistent. This is my mistake, sorry for it.